### ISSUE TITLE
Remove or replace ASCII art image in README

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-contrib/geofield#44]
The README starts with a nice ASCII art image which looks good on the Github page. It looks however broken on the backdropcms.org project page (and in the project browser).

Screenshots:

**Github:**
<img src="https://github.com/backdrop-contrib/geofield/assets/14188307/d3de9233-7d4a-4c25-9362-cc05c5c7a4cb" width="360">

**backdropcms.org:**
<img src="https://github.com/backdrop-contrib/geofield/assets/14188307/6ffebaf6-d340-4bb7-a864-b794228f81e4" width="540">

**Comment by indigoxela** [comment]
I totally agree, this really has to go, a PR's available: #50 That fake image provides no value and looks awkward on backdropcms.org.
